src/Leaderboard.ts

```diff
@@ -183,6 +183,7 @@ export default class Leaderboard {
         }.json`,
         headers: {
           Cookie: `session=${process.env.AOC_SESSION}`,
+          'User-Agent': 'github.com/TrojanerHD/AdventofCode-Bot by mail@trojaner.dev'
```

Comment:
This seems good, however, I think it would be nicer if we added the `https://`, it isn't necessary tho.

```suggestion
          'User-Agent': 'https://github.com/TrojanerHD/AdventofCode-Bot by mail@trojaner.dev'
```

Reply:
In the linked reddit article the provided example was without the `https://` so I was not sure if it was left out by purpose

Reply:
I don't think they spent a lot of time on that. It probably doesn't matter and comes down to pure preference.
I personally would use the `https://` prefix just to make it `ctrl+click`-able in terminal emulators and text editors.